### PR TITLE
Function refactoring

### PR DESCRIPTION
**Pull request by zdj6373**
getContainersHelper:The function returns is "return resp, err".
From the realization of function ListContainers,Resp and err must have one for nil , so this is no problem ; 
but the middle judge is not necessary. So remove it.

### CODE REVIEW

File: pkg/kubelet/kuberuntime/kuberuntime_container.go [hunk 1]
```diff
@@ -358,9 +358,6 @@ func (m *kubeGenericRuntimeManager) getKubeletContainers(allContainers bool) ([]
 // getContainers lists containers by filter.
 func (m *kubeGenericRuntimeManager) getContainersHelper(filter *runtimeapi.ContainerFilter) ([]*runtimeapi.Container, error) {
 	resp, err := m.runtimeService.ListContainers(filter)
-	if err != nil {
-		return nil, err
-	}
```
Comment (xiangpengzhao) [inline pkg/kubelet/kuberuntime/kuberuntime_container.go]: Seems like we can `return m.runtimeService.ListContainers(filter)` directly ?

Comment (derekwaynecarr) [reply pkg/kubelet/kuberuntime/kuberuntime_container.go]: +1

Comment (zdj6373) [reply pkg/kubelet/kuberuntime/kuberuntime_container.go]: already edited

**Comment by zdj6373** [comment]
keep it open another 90 days

**Pull request closed without merging**